utils/anaInfo.py

```diff
@@ -17,6 +17,12 @@
 scurveMeanMin = 0.1 #: points are removed if they satisfy scurveMean < scurveMeanMin
 scurveMeanFracErrMin = 0.001 #: points are removed if they satisfy scurveMeanError/scurveMean < scurveFracErrMin
 
+#: The default values for the cuts that determine the scurve fit quality masks
+maxEffPedPercentDefault=0.1
+highNoiseCutDefault=20
+deadChanCutLowDefault=0
+deadChanCutHighDefault=0
```

Comment:
Are these constants used anywhere else aside from `utils/anaoptions.py`?  If not remove.  Additionally these defaults do not match what was currently implemented.

Reply:
they are also used in utils/threshAlgos.py to set the arguments of the function calibrateThrDAC(...) if they are not provided

Reply:
the defaults are changed back to the ones currently implemented (I took the ones above from https://github.com/cms-gem-daq-project/sw_utils/blob/develop/scripts/calibrateArmDac.sh)